editor/icons/GizmoDecal.svg

```diff
@@ -0,0 +1 @@
+<svg height="128" viewBox="0 0 128 128" width="128" xmlns="http://www.w3.org/2000/svg"><g stroke-width="8.00001"><path d="m111.31322 55.933581a48.000003 48.000002 0 1 0 -71.280012 49.527999z" fill="#f7f5cf"/><path d="m111.31322 55.933581a48.000003 48.000002 0 0 0 -71.280012 49.527999z" fill="#e1b44c"/><path d="m40.033208 105.46158a48.000003 48.000002 0 0 0 71.280012-49.527999z" fill="#f7f5cf" fill-opacity=".4"/></g><ellipse cx="64" cy="64" rx="50" ry="50.000004" style="fill:none;stroke:#000;stroke-width:3.99999;stroke-linecap:round;stroke-linejoin:round;stroke-opacity:.3"/></svg>
```

Comment:
We should avoid `style` attributes, as they cannot be converted according to our colormap.

Reply:
Gizmo icons never have their colors converted (they are part of the 3D viewport which doesn't follow dark/light theme). I'd still like to get rid of those properties if possible, but I'm not sure how.